platform/logging/logger.go

```diff
@@ -2,13 +2,22 @@ package logging
 
 import (
 	"fmt"
+	"testing"
 	"time"
 
 	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
 )
 
 // Logger follow the standard go Logger and only give access to `Println`
-type Logger struct {
+type Logger interface {
+	Printf(string, ...interface{})
+	Println(...interface{})
+	PrintRequestResponse(RequestAttributes, TraceAttribute, DurationAttribute, RemoteAddressAttribute, string)
+	Sync() error
+}
```

Comment:
What is the point of that interface ?

Reply:
You are right, I thought the type of the logger would be different, but seems like it's fine, I will change it out

Reply:
yeah that was the whole point of using this lib :)

Reply:
Updated it :)